specification/security/query_api_keys/QueryApiKeysRequest.ts

```diff
@@ -17,10 +17,13 @@
  * under the License.
  */
 
+import { Dictionary } from '@spec_utils/Dictionary'
+import { APIKeyAggregationContainer } from 'APIKeyAggregationContainer'
```

Comment:
```suggestion
import { APIKeyAggregationContainer } from './types'
```

This file should be called `types`.